zizroc/villager
# Export the classes and move R files

Fixes issue #11 .

A couple of changes here

1. The parser didn't like having an R directory without any files in it. I checked other packages and found that many of them put their files in the root R directory so I made that change.

2. I added some other documentation and R complains a _little_ less now


To Test:

1. Check this branch out
2. In the upper right panel of RStudio click the 'Build' tab
3. Click 'Check', note that there aren't any errors (probably 1 warning and 1 note)


Note: Either use RStudio's Install Package button to install `villager` or use devtools to install the package from this github repository. In other R projects, you can now use `library(villager)`
